src/docs/03-concepts/10-http-api.md

```diff
@@ -5,208 +5,1201 @@ permalink: /docs/concepts/http-api
 
-Refer to instructions described here: https://github.com/uber/cadence/tree/master/docker#using-docker-image-for-production 
+Additionally add two more environment variables:
 
-Additionally add two more environment variables: 
 ```bash
 docker run
 <...>
-    -e FRONTEND_HTTP_PORT=8808                          -- HTTP PORT TO LISTEN 
+    -e FRONTEND_HTTP_PORT=8800                          -- HTTP PORT TO LISTEN 
     -e FRONTEND_HTTP_PROCEDURES=uber.cadence.api.v1.WorkflowAPI::StartWorkflowExecution  -- List of API methods exposed
     ubercadence/server:<tag> 
 ```
 
+### Using docker-compose
 
- 
-
-## Using docker-compose 
+Add HTTP environment variables to docker/docker-compose.yml configuration:
 
-Add HTTP environment variables to docker/docker-compose.yml configuration: 
 ```yaml
-cadence: 
-  image: ubercadence/server:master-auto-setup 
-  ports: 
-   - "8000:8000" 
-   - "8001:8001" 
-   - "8002:8002" 
-   - "8003:8003" 
-   - "7933:7933" 
-   - "7934:7934" 
-   - "7935:7935" 
-   - "7939:7939" 
-   - "7833:7833" 
-   - "8808:8808" 
-  environment: 
-    - "CASSANDRA_SEEDS=cassandra" 
-    - "PROMETHEUS_ENDPOINT_0=0.0.0.0:8000" 
-    - "PROMETHEUS_ENDPOINT_1=0.0.0.0:8001" 
-    - "PROMETHEUS_ENDPOINT_2=0.0.0.0:8002" 
-    - "PROMETHEUS_ENDPOINT_3=0.0.0.0:8003" 
-    - "DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development.yaml" 
-    - "FRONTEND_HTTP_PORT=8808" 
+cadence:
+  image: ubercadence/server:master-auto-setup
+  ports:
+    - "8000:8000"
+    - "8001:8001"
+    - "8002:8002"
+    - "8003:8003"
+    - "7933:7933"
+    - "7934:7934"
+    - "7935:7935"
+    - "7939:7939"
+    - "7833:7833"
+    - "8800:8800"
+  environment:
+    - "CASSANDRA_SEEDS=cassandra"
+    - "PROMETHEUS_ENDPOINT_0=0.0.0.0:8000"
+    - "PROMETHEUS_ENDPOINT_1=0.0.0.0:8001"
+    - "PROMETHEUS_ENDPOINT_2=0.0.0.0:8002"
+    - "PROMETHEUS_ENDPOINT_3=0.0.0.0:8003"
+    - "DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development.yaml"
+    - "FRONTEND_HTTP_PORT=8800"
     - "FRONTEND_HTTP_PROCEDURES=uber.cadence.api.v1.WorkflowAPI::StartWorkflowExecution" 
 ```
 
+## Using HTTP API
 
- 
+Start a workflow using curl command
+
+```bash
+curl -X POST http://0.0.0.0:8800 \
+    -H 'context-ttl-ms: 2000' \
+    -H 'rpc-caller: rpc-client-name' \
+    -H 'rpc-service: cadence-frontend' \
+    -H 'rpc-encoding: json' \
+    -H 'rpc-procedure: uber.cadence.api.v1.WorkflowAPI::StartWorkflowExecution' \
+    -d @data.json 
+```
 
-## Using HTTP API 
+Where `data.json` content looks something like this:
 
-Start a workflow using curl command 
+```json
+{
+  "domain": "sample-domain",
+  "workflowId": "workflowid123",
+  "execution_start_to_close_timeout": "11s",
+  "task_start_to_close_timeout": "10s",
+  "workflowType": {
+    "name": "workflow_type"
+  },
+  "taskList": {
+    "name": "tasklist-name"
+  },
+  "identity": "My custom caller identity",
+  "requestId": "4D1E4058-6FCF-4BA8-BF16-8FA8B02F9651"
+} 
+```
+
+Describe a cluster using curl command
```

Comment:
shouldn't this be under Admin API?

Reply:
It's also in Admin API section. This part only serves as an initial example. Do we want to remove it here and only show DescribeCluster in the Admin API section?